1-exercises/A-this/exercise-2.js

```diff
@@ -6,18 +6,20 @@
 
 */
 
-
 let person = {
-    name: "Maggie",
-    age: 26,
-    currentAddress: "London",
-    changeAddress: /* todo */
+  name: "Maggie",
+  age: 26,
+  currentAddress: "London",
+  changeAddress: function (newAddress) {
+    this.currentAddress = newAddress;
+  } /* todo */,
```

Comment:
This is a very nice and functional feature, however, you no longer have to include: and write functions when writing methods (functions inside objects).

    changeAddress (newAddress) {
    this.currentAddress = newAddress;    
    }
    
    This gives you same result.